src/BenchmarkDotNet/Jobs/Argument.cs

```diff
@@ -45,8 +45,34 @@ public MonoArgument(string value) : base(value)
     /// example: new MsBuildArgument("/p:MyCustomSetting=123")
     /// </summary>
     [PublicAPI]
+
+    /// <summary>
+    /// A specialized MSBuild argument that sets a property using a quoted, semicolon-separated list of values.
+    /// </summary>
+    public class MsBuildProperty : MsBuildArgument
+    {
+        public MsBuildProperty(string key, params string[] values)
+            : base($"/p:{key}=\"{string.Join(";", values)}")
```

Comment:
It seems that quotes need to be escaped to work cross-platform.
```suggestion
            : base($"/p:{key}=\\\"{string.Join(";", values)}\\\"")
```